.github/workflows/tacos_unlock.yml

```diff
@@ -89,78 +101,35 @@ jobs:
           ref: ${{inputs.tacos_gha_ref}}
           path: tacos-gha
       - name: Setup
-        uses: ./tacos-gha/.github/actions/setup
+        uses: ./tacos-gha/.github/actions/just-the-basics
+      - name: setup terraform versions
+        run: |
+          gha-set-env 'TF_VERSION' < "$(nearest-config-file .terraform-version)"
+          gha-set-env 'TERRAGRUNT_VERSION' < "$(nearest-config-file .terragrunt-version)"
+      - name: Setup Terragrunt
+        uses: autero1/action-terragrunt@v1.3.2
         with:
-          ssh-private-key: ${{ secrets.ssh-private-key }}
-          # We explicitly list the low-concern actions, during which users will
-          # recieve a higher level of access -- the identity of the original PR
-          # author. All other actions will receive lower access -- the identity
-          # of the person pushing the "Re-run" button.
-          #
-          # see doc/permissions.md
-          user: |-
-            ${{
-              true
-              && github.event_name == 'pull_request'
-              && (
-                  false
-                  || github.event.action == 'closed'
-                  || github.event.action == 'converted_to_draft'
-              )
-
-              && github.event.pull_request.user.login
-
-              || github.triggering_actor
-            }}
-
+          terragrunt_version: ${{env.TERRAGRUNT_VERSION}}
+      - name: Setup Terraform
+        uses: hashicorp/setup-terraform@v3
+        with:
+          terraform_wrapper: false
+          terraform_version: ${{ env.TF_VERSION }}
       - name: Unlock
         id: main
         run: |
-          tf-step-summary "TACOS Unlock" "$TACOS_GHA_HOME/"lib/tacos/unlock
-
-      - name: Save matrix result
-        # we need to show any errors to end-users
+          # release all tfstate locks currently held 
+          quietly \
+          echo "$SLICES" | jq -r '.[]' | ./tacos-gha/lib/ci/unlock
```

Comment:
This says `quietly echo`, meaning the middle third of `echo`'s output will be elided if it succeeds with more than 30 lines of output. I don't believe this is what you intended.